en/developer_guide/field_type.rst

```diff
@@ -0,0 +1,107 @@
+Field type
+==========
+
+Open Orchestra offers the possibility to create the Content Types from the back office.
+Further information about it is available in the `Content Types documentation`_.
+
+When creating a content type, you can add several attributes of different types.
+By default Open Orchestra offers few types
+(text line, date, email, hidden, integer, choice, media, money, rich test, text area).
```

Comment:
I think it lacks a sentence introducing the doc like "These page explains how to create new field types to extend the possibilities."